source/features/link-to-changelog-file.tsx

```diff
@@ -62,10 +57,8 @@ async function init(): Promise<void | false> {
 		'.subnav-links', // Tag list
```

Comment:
Can you make sure this still works on both pages?

Reply:
Before I create this pr I have tested all test urls. It all works fine.

<img width="592" alt="image" src="https://github.com/user-attachments/assets/ee4e2cb5-1452-44ae-bd06-783824269fb5">